git-obliterate: add page (#6099)
tldr-pages/tldr: pages/common/git-obliterate.md

# git obliterate

> > Remove a file from all history in a Git repository.

Reviewer: Hrm, good point. What about this?
```suggestion
> Remove a file from a Git repository and all its history.
```

> > Remove a file from all history in a Git repository.

Reviewer 2: I think this is a bit more elegant:
```suggestion
> Delete a file and erase its history from a Git repository:
```

> Part of `git-extras`.
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-obliterate>.

> - Remove all existence of a file including history:

Reviewer 2: How about:
```suggestion
- Delete files without a trace including their history:
```

Author: Yes I like that but is sounds wrong.
Here:

```suggestion
- Delete files and traces of that file:
```

The trace would be their history

Reviewer: Perhaps this will read better?
```suggestion
- Remove a file from the current repository's history:
```

Reviewer 2: Yet onother suggestion:
```suggestion
- Delete files and their history:
```

`git obliterate {{file_1 file_2 ...}}`

> - Remove existence of a file between 2 commits:

Reviewer: Perhaps this will read better?
```suggestion
- Remove a file between 2 commits in the current repository's history:
```

Author: I think not. It makes it sound a little worse IMO

Reviewer 2: I also prefer the original, however I have one small suggestion:
```suggestion
- Erase the existence of a file between 2 commits:
```

> `git obliterate {{file_1 file_2 ...}} -- {{commit_hash1}}..{{commit_hash2}}`

Reviewer 2: ```suggestion
`git obliterate {{file_1 file_2 ...}} -- {{commit_hash_1}}..{{commit_hash_2}}`
```
I don't know if it's really better, but at least it now looks the same as `file_1`.
